tests/natural_language/WriteGoodLintBearTest.py

```diff
@@ -0,0 +1,144 @@
+from bears.natural_language.WriteGoodLintBear import WriteGoodLintBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """The 70s era was awesome for music lovers.
+""".splitlines(keepends=True)
+
+bad_file = """So the 70s era was awesome for music lovers.
```

Comment:
can we change it to `so_file`? so it is consistent with the other files? and has more meaning as to why its `bad`?